apps/frontend/src/app/containers/createInsight/CreateInsight.tsx

```diff
@@ -40,43 +43,67 @@ export const CreateInsight = (): ReactElement => {
 
     const { users, loading, error } = useUserByCriterias(insight.criterias);
 
+    const [errorMessages, setErrorMesages] = useState<IInsightErrors>({
+        nameErrorMsg: '',
+        descriptionErrorMsg: '',
+        datesErrorMsg: [],
+        consentsErrorMsg: '',
+        candidatesErrorMsg: ''
+    });
+
     const handleSubmit = async () => {
         if (!id) throw new Error('This project does not exist.');
 
-        setPosting(true);
 
         const payload = {
             ...insight,
             project: { ...insight.project, id: parseInt(id) }
         };
 
-        const insightMutation = await createInsight(payload);
-
-        if (!insightMutation.response || insightMutation.isError) throw new Error('Failed to post insight.');
+        // I know this "any" is dirty, but i could not find insightMutation's type
+        let insightMutation: any = undefined;
```

Comment:
All mutation hooks return an object with the following format: 

`{response: Type, isLoading: boolean, isError: AxiosError}`

Unfortunately this is not defined as an interface, maybe it should be. :P